test/util/k8s/priority_class.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Thanks for adding E2E testing cases for this function.
However, I suggest we separate the test case code from the functionality.

If you want to create an E2E test case, please refer to this file. It contains the test case framework code.
https://github.com/vmware-tanzu/velero/blob/main/test/e2e/test/test.go

Please take this existing test case code as an example.
https://github.com/vmware-tanzu/velero/blob/main/test/e2e/resource-filtering/base.go